cli/azd/cmd/testdata/TestUsage-azd-env.snap

```diff
@@ -17,7 +17,7 @@ Available Commands
   refresh   	: Refresh environment settings by using information from a previous infrastructure provision.
   select    	: Set the default environment.
   set       	: Set one or more environment values.
-  set-secret	: Set a <name> as a reference to a Key Vault secret in the environment.
+  set-secret	: Set a `<name>` as a reference to a Key Vault secret in the environment.
```

Comment:
Do you mind sharing some more context around this change? I noticed some unescaped `<` and `>` characters above (`<environment-name>`) - does that need to be changed too?

Reply:
I see, so this piece of text?

<img width="681" height="578" alt="image" src="https://github.com/user-attachments/assets/fc46249a-c5ce-485e-bd70-bbcc07e51022" />

We could also just drop the angle brackets in the description and have it say "Set a name as a reference..." - I don't think it loses any clarity